libr/include/r_print.h

```diff
@@ -46,7 +46,7 @@ typedef struct r_print_t {
 	int datezone;
 	int (*write)(const unsigned char *buf, int len);
 	void (*cb_printf)(const char *str, ...);
-	const char *(*cb_color)(int idx, int last, bool bg);
+	char *(*cb_color)(int idx, int last, bool bg);
```

Comment:
Indont think we want heap allocated colors . Char* implies that the one who receives the pointer owns it and must free it and is able to modify it. So nope